### PR TITLE
Testing/update tests

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [X] Fix
- [ ] Feature
- [ ] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Minor changes to improves tests/descriptions/images.

### What ticket(s) do the changes resolve?
N/A
Please Link Issues to this PR as necessary.

### CODE REVIEW

File: cypress/integration/single-planet.js [hunk 1]
```diff
@@ -42,27 +42,27 @@ describe('Single Planet Testing', () => {
       .get('.gravity-words').should('contain', '354 pounds')
     cy.visit(`${baseURL}venus`)
     cy.get('input[name="weight"]').clear().type('9999999')
-      .get('.gravity-words').should('contain', '9044559 pounds')
+      .get('.gravity-words').should('contain', '9,044,559 pounds')
```
Comment (knees4bees) [inline cypress/integration/single-planet.js:45]: Good catch!

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const parseMass = (value: number, exponent: number) => {
 
 
 class App extends React.Component<{}, AllData> {
-  constructor(props: any) {
+  constructor(props: {}) {
```
Comment (knees4bees) [inline src/components/App/App.tsx:20]: Can't believe we missed this one! Thanks for fixing it.

**Review by knees4bees — APPROVED** [review]
Thanks for making these fixes, Connor.

File: cypress/integration/single-planet.js [hunk 3]
```diff
@@ -42,27 +42,27 @@ describe('Single Planet Testing', () => {
...
   })
   it('Should only allow numbers to be entered', () => {
     cy.visit(`${baseURL}neptune`)
       .get('.gravity-words').should('contain', '114 pounds')
     cy.get('input[name="weight"]').clear().type('134as2f')
-      .get('.gravity-words').should('contain', '1526 pounds')
+      .get('.gravity-words').should('contain', '1,526 pounds')
     cy.get('input[name="weight"]').clear().type('1342')
-      .get('.gravity-words').should('contain', '1526 pounds')
+      .get('.gravity-words').should('contain', '1,526 pounds')
```
Comment (alexthompson207) [inline cypress/integration/single-planet.js:55]: Nice job making these pass!

File: src/components/App/App.tsx (hunk 2 above)
Comment (alexthompson207) [reply src/components/App/App.tsx:20]: Likewise!

**Review by alexthompson207 — APPROVED** [review]

**Pull request merged**